src/payment/cart/Cart.jsx

```diff
@@ -64,7 +64,7 @@ class Cart extends React.Component {
             <SummaryTable price={summaryPrice} />
           )}
 
-          <Offers discounts={summaryDiscounts} offers={offers} />
+          <Offers discounts={summaryDiscounts} offers={offers} productsLength={products.length} />
```

Comment:
Suggestion we discussed:

`<Offers discounts={summaryDiscounts} offers={offers} isBundle={products.length > 1} />`